cmake/CreateBazelConfig.cmake

```diff
@@ -115,13 +115,14 @@ function (export_list_to_bazel filename VAR)
         APPEND "${filename}"
         [=[
 """Automatically generated unit tests list - DO NOT EDIT."""
```

Comment:
Probably not in this PR, but: `s/unit tests list/list(s)/` seems more appropriate

Reply:
Ack.